target/linux/ramips/image/mt7621.mk

```diff
@@ -343,6 +343,15 @@ define Device/dlink_dir-2660-a1
 endef
 TARGET_DEVICES += dlink_dir-2660-a1
 
+define Device/dlink_dir-853-r1
+$(Device/dlink_dir-8xx-r1)
+  DEVICE_MODEL := DIR-853
+  DEVICE_VARIANT := R1
+  DEVICE_PACKAGES += kmod-usb3 kmod-usb-ledtrig-usbport
+  IMAGE/factory.bin := append-kernel | append-rootfs | check-size
```

Comment:
Is this for "Emergency Room"? If so, can't sysupgrade image be used directly instead of this factory image?

Reply:
No. is here is instruction. Connect your client computer to LAN1 of the device
Set your client IP address manually to 192.168.0.101 / 255.255.255.0.
Then, power down the router, press and hold the reset button, then
re-plug it. Keep the reset button pressed until the internet LED stops
flashing
Call the recovery page or tftp for the device at http://192.168.0.1
Use the provided emergency web GUI to upload and flash a new firmware to
the device.

Reply:
Looks like @musashino205 is right here. The only difference between factory.bin and sysupgrade.bin is the metadata.

Reply:
https://github.com/openwrt/openwrt/pull/3468/commits/95f4b7400ab28d9f16af0f908cceef935ef83d2e
Sometimes it will not be possible to install from dlink frimware with this? 882 r1 webui is similar to dir 853 r1